Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-markdown/2.12.0-SNAPSHOT/jspwiki-markdown-2.12.0-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-markdown/2.11.0/jspwiki-markdown-2.11.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
+++  NEW CLASS: PUBLIC(+) org.apache.wiki.htmltowiki.syntax.markdown.MarkdownSyntaxDecorator  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 55.0 <- n.a.
	===  UNCHANGED SUPERCLASS: n.a.
	+++  NEW CONSTRUCTOR: PUBLIC(+) MarkdownSyntaxDecorator()
	+++  NEW METHOD: PUBLIC(+) void init(java.io.PrintWriter, java.util.Deque<java.lang.String>, java.util.Deque<java.lang.String>, org.apache.wiki.htmltowiki.WhitespaceTrimWriter, org.apache.wiki.htmltowiki.XHtmlToWikiConfig, org.apache.wiki.htmltowiki.XHtmlElementToWikiTranslator)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.ExternalLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ExternalLinkAttributeProviderState(org.apache.wiki.api.core.Context, boolean, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.ImageLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ImageLinkAttributeProviderState(org.apache.wiki.api.core.Context, java.lang.String, boolean)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.InterWikiLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC InterWikiLinkAttributeProviderState(org.apache.wiki.api.core.Context, boolean, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.JSPWikiLinkAttributeProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PROTECTED FINAL org.apache.wiki.parser.LinkParsingOperations linkOperations
	===  UNCHANGED FIELD: PROTECTED FINAL org.apache.wiki.api.core.Context wikiContext
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLinkAttributeProvider(org.apache.wiki.api.core.Context, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.ast.Node, com.vladsch.flexmark.html.renderer.AttributablePart, com.vladsch.flexmark.util.html.MutableAttributes)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.JSPWikiLinkAttributeProviderFactory  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLinkAttributeProviderFactory(org.apache.wiki.api.core.Context, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC com.vladsch.flexmark.html.AttributeProvider apply(com.vladsch.flexmark.html.renderer.LinkResolverContext)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalEditLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalEditLinkAttributeProviderState(org.apache.wiki.api.core.Context, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalFootnoteLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalFootnoteLinkAttributeProviderState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalFootnoteRefLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalFootnoteRefLinkAttributeProviderState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalLinkAttributeProviderState(org.apache.wiki.api.core.Context, boolean, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalReadLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalReadLinkAttributeProviderState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	GENERIC TEMPLATES: === T:com.vladsch.flexmark.util.ast.Node
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, com.vladsch.flexmark.util.ast.Node)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.WysiwygEditingAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WysiwygEditingAttributeProviderState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.MutableAttributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.AccessRuleLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC AccessRuleLinkNodePostProcessorState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.ExternalLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ExternalLinkNodePostProcessorState(org.apache.wiki.api.core.Context, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.ImageLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ImageLinkNodePostProcessorState(org.apache.wiki.api.core.Context, java.lang.String, boolean)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.InterWikiLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC InterWikiLinkNodePostProcessorState(org.apache.wiki.api.core.Context, com.vladsch.flexmark.util.ast.Document, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.JSPWikiLinkNodePostProcessor  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED FIELD: PROTECTED FINAL org.apache.wiki.api.core.Context m_context
	===  UNCHANGED FIELD: PROTECTED FINAL com.vladsch.flexmark.util.ast.Document document
	===  UNCHANGED FIELD: PROTECTED FINAL org.apache.wiki.parser.LinkParsingOperations linkOperations
	===  UNCHANGED FIELD: PROTECTED boolean m_useOutlinkImage
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLinkNodePostProcessor(org.apache.wiki.api.core.Context, com.vladsch.flexmark.util.ast.Document, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, com.vladsch.flexmark.util.ast.Node)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.JSPWikiNodePostProcessorFactory  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiNodePostProcessorFactory(org.apache.wiki.api.core.Context, com.vladsch.flexmark.util.data.DataHolder, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC com.vladsch.flexmark.parser.block.NodePostProcessor apply(com.vladsch.flexmark.util.ast.Document)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.LocalFootnoteLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalFootnoteLinkNodePostProcessorState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.LocalFootnoteRefLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalFootnoteRefLinkNodePostProcessorState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.LocalLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalLinkNodePostProcessorState(org.apache.wiki.api.core.Context, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.MetadataLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC MetadataLinkNodePostProcessorState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	GENERIC TEMPLATES: === T:com.vladsch.flexmark.util.ast.Node
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void process(com.vladsch.flexmark.util.ast.NodeTracker, com.vladsch.flexmark.util.ast.Node)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.PluginLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC PluginLinkNodePostProcessorState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.VariableLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC VariableLinkNodePostProcessorState(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.MarkdownForJSPWikiExtension  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC MarkdownForJSPWikiExtension(org.apache.wiki.api.core.Context, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
	===  UNCHANGED METHOD: PUBLIC void extend(com.vladsch.flexmark.html.HtmlRenderer$Builder, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void extend(com.vladsch.flexmark.parser.Parser$Builder)
	===  UNCHANGED METHOD: PUBLIC void parserOptions(com.vladsch.flexmark.util.data.MutableDataHolder)
	===  UNCHANGED METHOD: PUBLIC void rendererOptions(com.vladsch.flexmark.util.data.MutableDataHolder)
+++  NEW CLASS: PUBLIC(+) org.apache.wiki.markdown.MarkdownSetupEngineLifecycleExtension  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 55.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW CONSTRUCTOR: PUBLIC(+) MarkdownSetupEngineLifecycleExtension()
	+++  NEW METHOD: PUBLIC(+) void onInit(java.util.Properties)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.nodes.JSPWikiLink  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLink(com.vladsch.flexmark.ast.Link)
	===  UNCHANGED METHOD: PUBLIC java.lang.String getWikiLink()
	===  UNCHANGED METHOD: PUBLIC boolean hasRef()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toStringAttributes()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.renderer.JSPWikiLinkRenderer  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLinkRenderer()
	===  UNCHANGED METHOD: PUBLIC java.util.Set<com.vladsch.flexmark.html.renderer.NodeRenderingHandler<?>><com.vladsch.flexmark.html.renderer.NodeRenderingHandler<?>> getNodeRenderingHandlers()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.renderer.JSPWikiNodeRendererFactory  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiNodeRendererFactory(org.apache.wiki.api.core.Context)
	===  UNCHANGED METHOD: PUBLIC com.vladsch.flexmark.html.renderer.NodeRenderer apply(com.vladsch.flexmark.util.data.DataHolder)
	===  UNCHANGED METHOD: PUBLIC java.util.Set<java.lang.Class<?>><java.lang.Class<?>> getDelegates()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.parser.markdown.MarkdownDocument  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC MarkdownDocument(org.apache.wiki.api.core.Page, com.vladsch.flexmark.util.ast.Node)
	===  UNCHANGED METHOD: PUBLIC com.vladsch.flexmark.util.ast.Node getMarkdownNode()
	===  UNCHANGED METHOD: PUBLIC STATIC com.vladsch.flexmark.util.data.MutableDataSet options(org.apache.wiki.api.core.Context, boolean, java.util.List<org.apache.oro.text.regex.Pattern><org.apache.oro.text.regex.Pattern>)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.parser.markdown.MarkdownParser  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC MarkdownParser(org.apache.wiki.api.core.Context, java.io.Reader)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.parser.WikiDocument parse()
		===  UNCHANGED EXCEPTION: java.io.IOException
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.render.markdown.MarkdownRenderer  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC MarkdownRenderer(org.apache.wiki.api.core.Context, org.apache.wiki.parser.WikiDocument)
	===  UNCHANGED METHOD: PUBLIC java.lang.String getString()
		===  UNCHANGED EXCEPTION: java.io.IOException
